### ISSUE TITLE
(2.2.3 beta) Phone button prevents opening new conversation

### ISSUE THREAD

**Issue by maltemayer** [WhisperSystems/Signal-iOS#940]
It can happen that the phone button from a previous conversation stalls the plus button.

![signal-phone-plus-button](https://cloud.githubusercontent.com/assets/5073538/10977599/535b7132-83f2-11e5-86be-242e11691f90.png)

**Comment by FredericJacobs** [comment]
Do you have a way of reproducing this?

**Comment by maltemayer** [comment]
It happend again yesterday, I sent a text and a Image to an android user.  I was bounced back into the conversation when I tried to go to the overview.  When I went to the overview after that and it was like in the screenshot.
I just tried to reproduce it with an iPhone contact and the same android user under the same circumstances (see iOS animation bug as in #927 #929), but i couldn’t force it to happen again.
